buffer.c

```diff
@@ -559,151 +463,144 @@ static int array_buffer_view_has_dimension(zval *object, zval *zv_offset, int ch
 
 	if (check_empty) {
 		int retval;
-		zval *value = buffer_view_offset_get(intern, offset);
-		retval = zend_is_true(value);
-		zval_ptr_dtor(&value);
+		zval value;
+			    buffer_view_offset_get(intern, offset, &value);
+		retval = zend_is_true(&value);
```

Comment:
Could directly return here, without the extra variable.

The dtor here was dropped because it's always an int/double, so no dtor necessary?

Reply:
(Removed the extra variable)

Yes, the dtor was dropped because `value` lives in the stack, not in the heap.